syne_tune/backend/simulator_backend/simulator_backend.py

```diff
@@ -394,7 +328,7 @@ def _schedule(self, trial_id: int, config: Dict):
             StartEvent(trial_id=trial_id), event_time=time_start)
         self._debug_message('StartEvent', time=time_start, trial_id=trial_id,
                             pushed=True)
-        logger.info(f"Simulated time since start: {_time_start:.2f} secs")
+        logger.debug(f"Simulated time since start: {_time_start:.2f} secs")
```

Comment:
I'd leave this as info, this is a really important log output.

Reply:
Ok.